bin/polyserve

```diff
@@ -24,6 +24,7 @@ Options:
   -b <browsername> use this browser instead of default
                    (ex: 'Google Chrome Canary')
   -H <hostname>    use this hostname instead of localhost
+  --cors           enable permissive cors headers in server responses
```

Comment:
Would you consider it okay to use an optional extra parameter? `--cors` implies `*`, `--cors google.com` sets the allowed origin to `google.com`.